vendor/modules.txt

```diff
@@ -223,6 +223,8 @@ gopkg.in/alecthomas/kingpin.v3-unstable
 gopkg.in/inf.v0
 # gopkg.in/yaml.v2 v2.2.2
 gopkg.in/yaml.v2
```

Comment:
You still need this?

Reply:
I didnt make that file so I dont know, it could be any libs or other dependencies pulling it in, thats made by `go mod init/vendor`